src/Form.php

```diff
@@ -66,6 +66,26 @@ public function setName($name) {
 
 
     /**
+     * @param RequestContext $requestContext
+     * @return $this
+     */
+    public function handleRequestContext(RequestContext $requestContext) {
+      $this->cleanValidationFlag();
+      foreach ($this->getElements() as $element) {
+        $element->handleRequestContext($requestContext);
+      }
+
+      $this->data = $requestContext->all();
```

Comment:
If form contain 1 field `<input name="name" >` But user send 20 fields with post. Then this-data contains all post data. I think it should contains only 1 field `name`